ChurchCRM/CRM
# 2 bug problem saving from familyeditor

# Partial Fix to FamilyEditor save problem, Issue #6833 
Currently this just adjusts the quoting on the NULL references so that the SQL is properly formed.  This is done on 3 fields.

## No Screenshots 

## How to test the changes?
Prior to this fix, attempting to edit and save family info (FamilyEditor) with a blank for info on Latitude, Longitude, or Wedding Date, would result in a server error due to malformed SQL.  After this fix, any or all of those fields should be able to be blank.

## Type of change


# How Has This Been Tested?
Tested via editing a family, and leaving the above referenced fields blank, then trying to save.  The SQL error no longer occurs, but now there is a new Slim Application error related to getting photo thumbnail.  Details of this error have been added to #6833.  Since saving the family info is still not possible, I suggest the fixes for the Slim Application error be corrected within this same Issue/Pull Request.  Those changes have not yet been tracked down, but I'm initiating this pull request early to get feedback, and to allow others to try to recreate the new problem and possibly offer suggestions.  If it is desired to close this PR and open a new issue/PR for the Slim Application error, then this PR could be merged, and a new issue created.

# Checklist:
